tests/testthat/test-derive_locf_records.R

```diff
@@ -0,0 +1,100 @@
+test_that("derive_locf_records Test 1: new locf observations are derived correctly when visits are missing", {
+  library(tibble)
+  library(tidyr)
```

Comment:
This needs to be removed.  Use `tibble::function` or `tidyr::function` if you need something from these packages that is not available in our `admiral-package.R` file